examples/create-account-with-thresholdkey.js

```diff
@@ -0,0 +1,153 @@
+import {
+    Client,
+    PrivateKey,
+    AccountId,
+    Hbar,
+    AccountCreateTransaction,
+    TransferTransaction,
+    AccountBalanceQuery,
+    AccountDeleteTransaction,
+    KeyList,
+    Logger,
+    LogLevel,
+} from "@hashgraph/sdk";
+
+import dotenv from "dotenv";
+
+dotenv.config();
+
+// Set default log level to 'silent' if SDK_LOG_LEVEL is not specified in .env
+const SDK_LOG_LEVEL = process.env.SDK_LOG_LEVEL || "SILENT";
+
+// Initialize Logger with the specified log level from the environment variable
+const logger = new Logger(LogLevel._fromString(SDK_LOG_LEVEL));
```

Comment:
Could you please check how the setup is handled in the other JS examples? I'm encountering an error when trying to run this example, and it would be best not to introduce any additional environment configurations.

The error:
```
file:///Users/ivaylogarnev/Desktop/hedera-sdk-js/hedera-sdk-js/src/logger/LogLevel.js:79
                throw new Error(`Unknown log level: ${level}`);
                      ^

Error: Unknown log level: SILENT
    at LogLevel._fromString (file:///Users/ivaylogarnev/Desktop/hedera-sdk-js/hedera-sdk-js/src/logger/LogLevel.js:79:23)
```

Reply:
sorry about that my mistake.. made changes
you can test now should be working fine now